package.json

```diff
@@ -60,21 +60,21 @@
   },
   "dependencies": {
     "assert-plus": "^1.0.0",
-    "backoff": "^2.4.1",
-    "bunyan": "^1.8.3",
-    "fast-safe-stringify": "^2.0.3",
+    "backoff": "^2.5.0",
+    "bunyan": "^1.8.12",
+    "fast-safe-stringify": "^2.0.6",
     "keep-alive-agent": "0.0.1",
-    "lodash": "^4.17.10",
+    "lodash": "^4.17.11",
     "lru-cache": "^4.0.1",
```

Comment:
Might as well bump lru-cache to the same major as [restify](https://github.com/restify/node-restify/blob/f363b36b0e5176c3f134b345c057ec6199d70b83/package.json#L105)?